com.unity.render-pipelines.high-definition/Editor/Material/Unlit/ShaderGraph/HDUnlitGUI.cs

```diff
@@ -21,6 +21,11 @@ class HDUnlitGUI : HDShaderGUI
             new AdvancedOptionsUIBlock(MaterialUIBlock.Expandable.Advance, ~AdvancedOptionsUIBlock.Features.SpecularOcclusion)
         };
 
+        /// <summary>
+        /// Implement your custom GUI in this function.false You'll probably want to use the MaterialUIBlock to display a UI similar to HDRP shaders.
```

Comment:
Implement your custom GUI in this function. To display a UI similar to HDRP shaders, use a MaterialUIBlock.